packages/web/src/components/link/Link.module.css

```diff
@@ -9,3 +9,11 @@
   text-decoration: underline !important;
   text-decoration-color: var(--primary-light-2);
 }
+
+.root path {
```

Comment:
sure we want this for all `Link` components? probably, just double checking

Reply:
i'm not totally sure... it's a good point. i kind of figured if we're making the text pink already, any icon should go with it? i'll check w julian and dylan